common/thorhelper/thorxmlread.cpp

```diff
@@ -1653,9 +1654,22 @@ class CXMLParse : public CInterface, implements IXMLParse
         void setMarkingStream(CMarkReadBase &_marking) { marking.set(&_marking); }
         CXPath &queryXPath() { return xpath; }
 
-// IPTreeMaker
+        bool checkSkipJsonNode(const char *tag)
+        {
+            if (!isJson || stack.ordinality()) //json root level only
```

Comment:
minor: If just for special root tags, method name should perhaps be changed from checkSkipJsonNode to checkSkipJsonRoot or similar (since it's called for all)